### PR TITLE
Editing changes for readability, and added some new content. See file changes for more

### PR DESCRIPTION
**Pull request by remyb2canada**
- edited table of content
- added Private Sector bullet points (similar to Citizens)
- changed IITB-ESDC content
- added References section to lighten references throughout the document

### CODE REVIEW

**Comment by remyb2canada** [comment]
wait... there are still conflicts...

File: _pages/en/it-picture-long-term.md [hunk 1]
```diff
@@ -26,24 +27,24 @@ In the following sections, we're trying to take the same end state and create "v
 
 ### Citizens
 
-- Cyber security and privacy - citizens have a choice to share their personal data for convenience
+- Cyber security and privacy: citizens have a choice to share their personal data for convenience
 - Generational change: Citizens are used to deal with technologies, they do not face technical illiteracy like 2019.
+- One GC: Citizens do not interact with individual departments, they interact with federal government's services without having to understand the way those are administered.
```
Comment (jaysonmc) [inline _pages/en/it-picture-long-term.md]: Which looks like ... what? In 10 years? Through a government portal, or integrated with existing services? 

To quote from our meeting, "Tangible tangible."

File: _pages/en/it-picture-long-term.md [hunk 2]
```diff
@@ -59,6 +60,12 @@ Said service representatives would be able to...
 GC departments have common kiosks to offer services, such as passports, taxes, grants as well as benefits from ESDC.
 Outreach programs are deployed to raise awareness in low uptake areas.
 
+### Private Sector
+
+- Private sector includes for profit companies, not for profit organizations, academia, and private citizens
```
Comment (jaysonmc) [inline _pages/en/it-picture-long-term.md]: Not sure what this is intended to mean...?

File: _pages/en/it-picture-long-term.md [hunk 3]
```diff
@@ -99,11 +112,18 @@ EA teams provide a loosely coupled architecture (reusable infrastructure artifac
 EA is held accountable to:
 
 - Provide an artifact repository which, when used by other teams, puts them into compliance
-- Other teams may freely opt out of their services, and they will be measured against the uptake rate from their clients
```
Comment (jaysonmc) [inline _pages/en/it-picture-long-term.md]: I am putting this one back in ... 
Why are we taking out the one measureable item we have? We talk a lot about metrics, yet seem uncomfortable putting them in. This aligns well with service culture and pushing for the collection of metrics...

File: _pages/en/it-picture-long-term.md [hunk 4]
```diff
@@ -83,11 +91,16 @@ Ecosystem of services are created by leveraging private sector expertise where o
...
 
-- ESDC custom built software are published as open source, allowing Academia and learning institutions to use them, helping grow technological skill sets in the country, fostering economic growth, improving transparency and trust in government, and increasing the breadth of talent for ESDC to use by welcoming contributions back
+- ESDC custom built software are published as open source, allowing Academia and learning institutions to use them, helping grow technological skill sets in the country, fostering economic growth, improving transparency and trust in government, and increasing the breadth of talent for ESDC to use by welcoming contributions back ( e.g. The Canadian version of [code.gov](https://code.gov/))
 - ESDC data (performance and automated decision making statistics) published as open datasets
 
 ### IITB-ESDC
 
+- Technology is used in all of ESDC's service delivery
+- IT services are able to response in a timely manner to the ever growing pace of business change
+- The relationship between ESDC Service lines and IITB is harmonized where business understands the risk of IT and the success of IT is measured via business success
```
Comment (jaysonmc) [inline _pages/en/it-picture-long-term.md]: Not tangible ... I propose 
"- IT success is measured through business success"

I was going to just change it, but I wasn't exactly sure that's what you wanted to say.

File: _pages/en/it-picture-long-term.md [hunk 5]
```diff
@@ -99,11 +112,18 @@ EA teams provide a loosely coupled architecture (reusable infrastructure artifac
 EA is held accountable to:
 
 - Provide an artifact repository which, when used by other teams, puts them into compliance
-- Other teams may freely opt out of their services, and they will be measured against the uptake rate from their clients
+- Produce reference architectures for solution teams to use and align their designs to
```
Comment (jaysonmc) [inline _pages/en/it-picture-long-term.md]: We need to automate EA, not have them just produce documentation as they do now ... no?

File: _pages/en/it-picture-long-term.md [hunk 6]
```diff
@@ -99,11 +112,18 @@ EA teams provide a loosely coupled architecture (reusable infrastructure artifac
...
+- Recommend investment decision bodies on areas IT needs to invest in
```
Comment (jaysonmc) [inline _pages/en/it-picture-long-term.md]: Recommend? I would argue that's a bit of a fluff word. There's no accountability in 'recommendations' 

What do they do? With what authority?

File: _pages/en/it-picture-long-term.md (hunk 1 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: I thought that is tangible. It's the OneGC.
Citizens do not see ESDC. They see Federal Government.

File: _pages/en/it-picture-long-term.md (hunk 2 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: To clarify that private sector isn't limited to private companies. But that we will work with non profit, academia, and even private citizens

File: _pages/en/it-picture-long-term.md (hunk 4 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: Agree that #2 is a repeat of #3. Will remove it.
I think #1 (tech used in all services) is important to highlight. It's a statement that services cannot be delivered without tech.
#3 is a reference to War Peace IT. It's not just success, it's also risk.

File: _pages/en/it-picture-long-term.md (hunk 5 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: I disagreed, partly. The trend is to light reference architectures. I will look for references to support.
It's still DevOps. But devOps is using reference arch. Look at AWS and how many ref arch they produce to help developers use their services.

File: _pages/en/it-picture-long-term.md (hunk 1 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: will reword to clarify

File: _pages/en/it-picture-long-term.md (hunk 6 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: Removed. Until I am able to defend the statement :)
Changed it to be about adherence to compliance (which may require recommending reviewing certain policies and directives that are in conflict)

File: _pages/en/it-picture-long-term.md (hunk 2 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: See commit #02a1519 with rewording

File: _pages/en/it-picture-long-term.md (hunk 4 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: See commit #02a1519 with rewording

File: _pages/en/it-picture-long-term.md (hunk 5 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: See commit #02a1519 with wording changes to add DevOps and common vision

File: _pages/en/it-picture-long-term.md (hunk 6 above)
Comment (remyb2canada) [reply _pages/en/it-picture-long-term.md]: See commit #02a1519 with removal and wordings on compliance

**Pull request merged**